Application/EdFi.Ods.Standard/Security/Authorization/Overrides/DisciplineActionRelationshipsAuthorizationContextDataProvider.cs

```diff
@@ -0,0 +1,61 @@
+﻿// SPDX - License - Identifier: Apache - 2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System;
+using System.Diagnostics.CodeAnalysis;
+using EdFi.Ods.Api.Security.AuthorizationStrategies.Relationships;
+using EdFi.Ods.Entities.Common.EdFi;
+using EdFi.Ods.Entities.NHibernate.DisciplineActionAggregate.EdFi;
+
+namespace EdFi.Ods.Standard.Security.Authorization.Overrides
+{
+    /// <summary>
+    /// Creates and returns an <see cref="RelationshipsAuthorizationContextData"/> instance for making authorization decisions for access to the edfi.DisciplineAction table of the DisciplineAction aggregate in the Ods Database.
+    /// </summary>
+    [ExcludeFromCodeCoverage]
+    public class DisciplineActionRelationshipsAuthorizationContextDataProvider<TContextData> : IRelationshipsAuthorizationContextDataProvider<IDisciplineAction, TContextData>
```

Comment:
Any reason not to register this override in `RelationshipsAuthorizationContextDataProviderOverridesModule.cs`? (like we do in main and main-6x branches).

I manually registered it to test it, and got the next error while trying to get the DisciplineActions:
`{
  "message": "Unable to authorize the request because there is no authorization support for associating the API client's associated claim values (of 'SchoolId') with the requested resource ('DisciplineAction')."
}`.

Reply:
Added only to use the DisciplineAction scenario; can be removed from this commit and after that we can discuss bringint ODS-5459 to 5.x